backend/api/admin/src/test/java/co/airy/core/api/admin/ChannelsControllerTest.java

```diff
@@ -76,8 +76,8 @@ static void afterAll() throws Exception {
 
     private static boolean testDataInitialized = false;
 
-    final static String facebookToken = "token";
-    final static Channel connectedChannel = Channel.newBuilder()
+    static final String facebookToken = "token";
```

Comment:
TIL that there is a suggested order for these in the Java spec

Reply:
yes!